iOS_SDK/OneSignalSDK/Source/OneSignalNotificationServiceExtensionHandler.m

```diff
@@ -41,7 +41,7 @@ @implementation OneSignalNotificationServiceExtensionHandler
 + (UNMutableNotificationContent*)didReceiveNotificationExtensionRequest:(UNNotificationRequest*)request
                                          withMutableNotificationContent:(UNMutableNotificationContent*)replacementContent {
     // Set default log level of NSE to VERBOSE so we get all logs from NSE logic
-    [OneSignal setLogLevel:ONE_S_LL_VERBOSE visualLevel:ONE_S_LL_NONE];
+    //[OneSignal setLogLevel:ONE_S_LL_VERBOSE visualLevel:ONE_S_LL_NONE];
```

Comment:
is this ok?

Reply:
We need to do this or the unit tests print too much and break CircleCI

Reply:
gotcha, but I think we want to keep this setLogLevel, Travis will print verbose not debug, so if there are logs that were added maybe change to debug?

Reply:
This is the line where we are setting the log level for the tests.
https://github.com/OneSignal/OneSignal-iOS-SDK/blob/2e4a36cf7dc2edd3ffcf525d9f28edaf0011ad72/iOS_SDK/OneSignalSDK/UnitTests/UnitTestCommonMethods.m#L156

The line noted in this specific comment is in the NSE, which depending on the run order of the tests would turn up the log level.

At first I thought this was our dev app but this is our SDK source. Let's remove this line as the comment above it. I think log level setting is best to live in the NSE code in our setup guide.

Reply:
Removed this line and updated the verbose message below it. Also updated the iOS setup